README.md

```diff
@@ -21,6 +21,7 @@ Table of Contents
             * [Server Response Lookup](#server-response-lookup)
          * [Protocol Specification](#protocol-specification)
          * [Session and Transaction Delay Specification](#session-and-transaction-delay-specification)
+         * [HTTP/2 Transaction Abort](#http2-transaction-abort)
```

Comment:
We talked about this offline, but just to record it here:

* Let's add an HTTP/2 specification under HTTP Specification and describe the frame syntax there.
* Let's add what frames are supported currently (HEADERS, DATA, RST_STREAM), with a section for each. The HEADERS and DATA are super simple and should be short - they simply look like the format described for HTTP/1 above.
* The RST_STREAM is more thorough and we'll include an example. (You already have this below...so this is mostly just a re-organization request.)